tests/default/_core/point_in_time/all.yaml

```diff
@@ -0,0 +1,33 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test point-in-time.
+version: '>= 2.4'
+epilogues:
+  - path: /_search/point_in_time/_all
```

Comment:
I have a little question. Is it okay to duplicate the use of the API
`DELETE /_search/point_in_time/_all?`
Does this ensure that we will definitely delete it?

Reply:
This is just cleanup/safeguard in case the test fails.